src/catalog/default/default_functions.cpp

```diff
@@ -156,6 +156,10 @@ static DefaultMacro internal_macros[] = {
 	// date functions
 	{DEFAULT_SCHEMA, "date_add", {"date", "interval", nullptr}, "date + interval"},
 
+	// convert min/max to arg_min/arg_max for collations
+	{DEFAULT_SCHEMA, "min", {"a", nullptr}, "arg_min(a, a)"},
```

Comment:
This will degrade performance and seems excessive just to get collations to work. This needs to be handled by min/max themselves (e.g. by transforming into arg_min/arg_max only in this scenario).

Reply:
You're right. Fix it by replacing min/max only in collation scenarios.